File: _includes/syntax_samples/if.rb
# standard
if condition
  "this"
else
  "that"
end

# else if
if condition
  "this"
elsif another_condition
  "this2"
else
  "that"
end

# one-liner
if condition then "this" else "that" end

# return value
return_val = if condition then "this" else "that" end
# also supports ternary operator
return_val = condition ? "this" : "that"

# unless 
return_val = unless condition then "that" else "this" end

# statement modifiers
return_val = "this" if condition

File: _includes/syntax_samples/if.cs
// one liner ifs/elses don't require curly braces

// standard
if (condition)
{
  result = "this";
}
else
{
  result = "that";
}

// else if
if (condition)
{
  result = "this";
}
else if (another_condition)
{
  result = "this2";
}
else
{
  result = "that";
}

// one-liner
if (condition) result = "this"; else result = "that";

// return value supported via ternary operator
result = condition ? "this" : "that";

// unless not supported
